### PR TITLE
Feature/reddit

### PR DESCRIPTION
**Pull request by jessicafarias**
We created 3 models which created User, Post and Comment classes inheriting from ApplicationRecord and then generating the ActiveRecord:: Migration[6.0] for each class.
We then added ActiveRecord validations :
- to verify the presence of the users in each post and comment
- to verify the presence of the email and username

### CODE REVIEW

File: app/models/comment.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  validates :user, presence: true
+  validates :post, presence: true
```
Comment (Dandush03) [inline app/models/comment.rb:5]: #### Missing Validation
- [x] `body` - Should not be able to receive an empty value. Missing Validation

File: app/models/post.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  validates :body, length: { maximum: 200 }
+  validates :user, presence: true
```
Comment (Dandush03) [inline app/models/post.rb]: #### Missing Validation
- [x] `title` - Should not be able to receive an empty value. Missing Validation
- [x] `body` - Should not be able to receive an empty value. Missing Validation

**Review by Dandush03 — CHANGES_REQUESTED** [review]
# Status: :red_circle:  Require Changes :red_circle: 
## Great Work so Far
### There are some validation that are missing, please check my comment below
# Happy Coding :rocket: :computer:

**Review by uryelah — APPROVED** [review]
Hi **Jessica Farías Rosado & Ara Camarillo Estudillo**, I'm Sarah your reviewer for this PR 👩‍🚀 

You did an awesome job with the validations, specially the one for the emal!

Go forth and rock your next project!  👩‍🎤  🎸  

-------

## **Your project is approved! 🎊 🥅 💛**

**You may close this Pull Request and merge the changes**


---------

    
    
**Keep being awesome devs and good luck with your next project!**

---------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**